pkg/frontend/load_test.go

```diff
@@ -616,7 +618,7 @@ func Test_rowToColumnAndSaveToStorage(t *testing.T) {
 			}
 			// XXX Vecs[0]?   What are we testing?
 			handler.batchData.Vecs[0] = vector.PreAllocType(Oid[i].ToType(), curBatchSize, curBatchSize, testutil.TestUtilMp)
-			convey.So(rowToColumnAndSaveToStorage(handler, force, row2colChoose), convey.ShouldNotBeNil)
+			convey.So(rowToColumnAndSaveToStorage(handler, proc, force, row2colChoose), convey.ShouldNotBeNil)
```

Comment:
you can add more a check here, to check if the load_test has clean all the memory or not.
`require.Equal(t, proc.mp.now(), proc.mp.old())`
(before that, maybe you need to clean up some bat by yourself, I'm not sure

Reply:
ok